src/containers/Elements/NodeProvider.js

```diff
@@ -60,30 +66,53 @@ class NodeProvider extends Component {
     const {
       interaction,
       nodes,
+      nodeColor,
+      showPanelAlways,
+      currentIds,
+      isOrigin,
     } = this.props;
 
     const label = node => `${node.nickname}`;
     const selected = node => isMatch(node, this.props.activePromptAttributes);
+    const nodeType = isOrigin ? null : 'EXISTING_NODE';
```

Comment:
Seems like a work around, does accepted _type_ actually change?

Reply:
So this could be done differently, I'm sure. I toggled the type only because you may have one node that can be dragged to a side panel, and another right next to it that cannot. It seemed clunky to add and remove an entire zone based on which node is being dragged. Maybe there should be a new field though or something to toggle whether to allow a node to be dropped or not.

Reply:
Are we talking about hover functionality here? Dropping a node on a droppable zone doesn't do anything in itself, so this shouldn't cause any issues with still accepting a type even if it's not wanted (the dropping component itself can just do a noop).

Reply:
I hadn't considered a noop. I think the hover functionality can be handled elsewhere; I was just trying to get the active zones to be 0. I'll look and see if the dropping component can figure out when to do a noop.